### PR TITLE
CI(actions): Update actions to create one workflow per event name

### PR DESCRIPTION
**Pull request by titom73**
## Change Summary

Split actions into multiple workflow files based on event-name

- pull-request management
	- Molecule
	- check links
- push to devel
	- Send webhook to update docker
- PR triage

Goal of this PR is not to change all testing method, but just make workflows easier to maintain and avoid complex `if` statement in each step.

## Related Issue(s)

None

## Component(s) name

Github Actions

## Proposed changes

- Rename files
- Merge some workflows
- Avoid as much as possible multiple event names

## How to test

Check PR https://github.com/titom73/ansible-avd/pull/24 

### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: .github/workflows/pull-request-triage.yml [hunk 1]
```diff
@@ -9,7 +9,42 @@ on:
...
+  assign_author:
+    name: "Assign Author to PR"
+    # https://github.com/marketplace/actions/auto-author-assign
+    runs-on: ubuntu-latest
+    steps:
+      - uses: toshimaru/auto-author-assign@v1.2.0
+        with:
+          repo-token: "${{ secrets.GITHUB_TOKEN }}"
+  pr_reviewvers:
+    name: "Assign PR reviewver"
```
Comment (ClausHolbechArista) [inline .github/workflows/pull-request-triage.yml]: typo, reviewer

Comment (carlbuchmann) [reply .github/workflows/pull-request-triage.yml]: resolved by 209de32

**Review by carlbuchmann — APPROVED** [review]
LGTM!

**Review by ClausHolbechArista — APPROVED** [review]
LGTM

**Pull request merged**